Refs: [HEAD], v5.1.0-1-g15196eb5
Author:     Eric D. Helms <ericdhelms@gmail.com>
AuthorDate: Mon Dec 14 14:52:36 2020 -0500
Commit:     Eric D. Helms <ericdhelms@gmail.com>
CommitDate: Mon Dec 14 14:52:36 2020 -0500

    Drop inline trailing comment due to RDoc bug https://bugs.ruby-lang.org/issues/14617
---
 lib/puma/control_cli.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/puma/control_cli.rb b/lib/puma/control_cli.rb
index df749ff6..4b015c6f 100644
--- a/lib/puma/control_cli.rb
+++ b/lib/puma/control_cli.rb
@@ -171,7 +171,7 @@ module Puma
           OpenSSL::SSL::SSLSocket.new(
             TCPSocket.new(uri.host, uri.port),
             OpenSSL::SSL::SSLContext.new)
-            .tap { |ssl| ssl.sync_close = true }  # default is false
+            .tap { |ssl| ssl.sync_close = true }
             .tap(&:connect)
         when 'tcp'
           TCPSocket.new uri.host, uri.port
